commands/go-mod-tidy.go

```diff
@@ -0,0 +1,18 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var goModTidy = &cobra.Command{
+	Use:   "go-mod-tidy",
+	Short: "Prune no-longer required commits from go.mod",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		runTool("go", append([]string{"mod", "tidy"}))
```

Comment:
Running `pre-commit run --all-files` on the repo itself complains about this line:

```
commands\go-mod-tidy.go:12:17: SA4021: x = append(y) is equivalent to x = y (staticcheck)
                runTool("go", append([]string{"mod", "tidy"}))
```

Compared to many of the other commands the main difference is that it does not include `args...`. It does not seem `go mod tidy` currently accepts all that many arguments, but there is *an* argument it accepts, and we could support it and future arguments by including `args...`.

Changing the line to:

```
runTool("go", append([]string{"mod", "tidy"}, args...))
```

Should fix the issue.